### PR TITLE
Changed date fields to datetime fields

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
This PR changes date fields (activation_date and expiration_date) to datetime python objects. This because we need to know the exact moment we want to activate or deactivate a tag

The datetime format will be discussed in other PR

### CODE REVIEW

File: eox_tagging/migrations/0001_initial.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# Generated by Django 1.11.29 on 2020-06-18 19:35
+# Generated by Django 1.11.29 on 2020-06-29 13:35
```
Comment (andrey-canon) [inline eox_tagging/migrations/0001_initial.py:2]: please don't modify previous migrations files which have already been executed

Comment (mariajgrimaldi) [reply eox_tagging/migrations/0001_initial.py]: I added a new migration for the changes. Thank you!

File: eox_tagging/test/test_viewset.py [hunk 2]
```diff
@@ -68,13 +68,50 @@ def test_create_tag(self):
...
+            "tag_type": "example_tag_1",
+            "tag_value": "example_tag_value",
+            "target_type": "user",
+            "target_id": "user_test",
+            "owner_type": "user",
+            "access": "PRIVATE",
+            "expiration_date": "2020-12-04T10:20:30.15785",
+        }
+
+        response = self.client.post("/api/v1/tags/", data, format='json')
```
Comment (andrey-canon) [inline eox_tagging/test/test_viewset.py:92]: reverse

File: eox_tagging/test/test_viewset.py [hunk 3]
```diff
@@ -68,13 +68,50 @@ def test_create_tag(self):
...
+
+        self.assertEqual(response.status_code, 201)
+
+    def test_create_tag_with_wrong_datetime_format(self):
+        """"
+        Used to test creating a tag using wrong format in datetime fields. This results in a
+        Bad request.
+        t"""
```
Comment (andrey-canon) [inline eox_tagging/test/test_viewset.py:100]: t of thank  you ?

Comment (mariajgrimaldi) [reply eox_tagging/test/test_viewset.py:100]: hahaha upsi. I'll fix that

File: eox_tagging/test/test_viewset.py (hunk 2 above)
Comment (mariajgrimaldi) [reply eox_tagging/test/test_viewset.py:92]: I fixed this in #16, should I rebase this PR with that one?

Comment (andrey-canon) [reply eox_tagging/test/test_viewset.py:92]: that pr was already approved, please squash, merge and rebase this one

Comment (mariajgrimaldi) [reply eox_tagging/test/test_viewset.py:92]: Right. Thank you :)

**Pull request merged**